solidity/contracts/OngoingAirdrops.sol

```diff
@@ -37,7 +37,39 @@ contract OngoingAirdrops is AccessControl, IOngoingAirdrops {
     bytes32 _campaign,
     bytes32 _root,
     TokenAmount[] calldata _tokensAllocation
-  ) external override {}
+  ) external override onlyRole(ADMIN_ROLE) {
+    if (_campaign == bytes32(0)) revert InvalidCampaign();
+    if (_root == bytes32(0)) revert InvalidMerkleRoot();
+    if (_tokensAllocation.length == 0) revert InvalidTokenAmount();
+
+    for (uint256 i = 0; i < _tokensAllocation.length; i++) {
```

Comment:
Is it cheaper if we don't initialize i?

Reply:
As said on TG. We had done previous tests that showed not assigning ` = 0` was an optimization... But new tests show it doesn't change. As always, I'm confused about the optimizer.